File: Back End/GiftShopAPI/GiftShopUser.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using GiftShopAPI.models;

namespace GiftShopAPI
{
    public class GiftShopUser
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public UserType UserType { get; set; }

    }
}

File: Front End/GiftShopUI/src/app/models/giftShopUser.ts
export interface GiftShopUser {
  id?: string;
  firstName?: string;
  lastName?: string;
  email?: string;
  password?: string;
  userType?: UserType;
}

export enum UserType {
  Unregistered = 0,
  Registered = 1,
  Admin = 2,
}
